backend/dataall/cdkproxy/stacks/pivot_role.py

```diff
@@ -0,0 +1,822 @@
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        's3:GetAccessPoint',
+                        's3:GetAccessPointPolicy',
+                        's3:ListAccessPoints',
+                        's3:CreateAccessPoint',
+                        's3:DeleteAccessPoint',
+                        's3:GetAccessPointPolicyStatus',
+                        's3:DeleteAccessPointPolicy',
+                        's3:PutAccessPointPolicy',
+                    ],
+                    resources=[f'arn:aws:s3:*:{self.account}:accesspoint/*'],
+                ),
+                # S3 Managed Buckets
+                iam.PolicyStatement(
+                    sid='ManagedBuckets',
+                    effect=iam.Effect.ALLOW,
+                    actions=['s3:List*', 's3:Delete*', 's3:Get*', 's3:Put*'],
+                    resources=[f'arn:aws:s3:::{env_resource_prefix}*'],
+                ),
+                # S3 Imported Buckets
+                iam.PolicyStatement(
+                    sid='ImportedBuckets',
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        's3:List*',
+                        's3:GetBucket*',
+                        's3:GetLifecycleConfiguration',
+                        's3:GetObject',
+                        's3:PutBucketPolicy',
+                        's3:PutBucketTagging',
+                        's3:PutObject',
+                        's3:PutObjectAcl',
+                        's3:PutBucketOwnershipControls',
+                    ],
+                    resources=['arn:aws:s3:::*'],
+                ),
+                # AWS Logging Buckets
+                iam.PolicyStatement(
+                    sid='AWSLoggingBuckets',
+                    effect=iam.Effect.ALLOW,
+                    actions=['s3:PutBucketAcl', 's3:PutBucketNotification'],
+                    resources=[f'arn:aws:s3:::{env_resource_prefix}-logging-*'],
+                ),
+                # Read Buckets
+                iam.PolicyStatement(
+                    sid='ReadBuckets',
+                    effect=iam.Effect.ALLOW,
+                    actions=['s3:ListAllMyBuckets', 's3:GetBucketLocation', 's3:PutBucketTagging'],
+                    resources=['*'],
+                ),
+                # CloudWatch Metrics
+                iam.PolicyStatement(
+                    sid='CWMetrics',
+                    effect=iam.Effect.ALLOW,
+                    actions=['cloudwatch:PutMetricData', 'cloudwatch:GetMetricData', 'cloudwatch:GetMetricStatistics'],
+                    resources=['*'],
+                ),
+                # Logs
+                iam.PolicyStatement(
+                    sid='Logs',
+                    effect=iam.Effect.ALLOW,
+                    actions=['logs:CreateLogGroup', 'logs:CreateLogStream', 'logs:PutLogEvents'],
+                    resources=[
+                        f'arn:aws:logs:*:{self.account}:/aws-glue/*',
```

Comment:
should be:
`    f"arn:aws:logs:*:{self.account}:log-group:/aws-glue/*",
                        f"arn:aws:logs:*:{self.account}:log-group:/aws/lambda/*",
                        f"arn:aws:logs:*:{self.account}:log-group:/{env_resource_prefix}*",
                    ],`